samcli/lib/pipeline/bootstrap/environment.py

```diff
@@ -138,7 +140,7 @@ def bootstrap(self, confirm_changeset: bool = True) -> bool:
 
         if self.did_user_provide_all_required_resources():
             click.secho(
-                f"\nAll required resources for the {self.name} environment exist, skipping creation.", fg="yellow"
+                self.color.yellow(f"\nAll required resources for the {self.name} environment exist, skipping creation.")
```

Comment:
Just curious are there any differences between using `self.color.yellow` and `fg="yellow"`?

Reply:
No difference in the functionality. Just cleaner code.